src/panels/config/storage/storage-breakdown-chart.ts

```diff
@@ -0,0 +1,293 @@
+              label: html`${this.hass.localize(
+                  `ui.panel.config.storage.segments.${child.id}`
+                ) ||
+                child.label ||
+                child.id}
+                <span style="color: var(--secondary-text-color)"
+                  >${roundWithOneDecimal(space)} GB</span
+                >`,
+            });
+          }
+        });
+      } else {
+        segments.push({
+          value: usedSpaceGB,
+          color: "var(--primary-color)",
+          label: html`${this.hass.localize(
+              "ui.panel.config.storage.segments.used"
+            )}
+            <span style="color: var(--secondary-text-color)"
+              >${roundWithOneDecimal(usedSpaceGB)} GB</span
+            >`,
+        });
+      }
+
+      segments.push({
+        value: freeSpaceGB,
+        color:
+          "var(--ha-bar-background-color, var(--secondary-background-color))",
+        label: html`${this.hass.localize(
+            "ui.panel.config.storage.segments.free"
+          )}
+          <span style="color: var(--secondary-text-color)"
+            >${roundWithOneDecimal(freeSpaceGB)} GB</span
+          >`,
+      });
+
+      return segments;
+    }
+  );
+
+  private _transformToSunburstData = memoizeOne(
+    (storageInfo: HostDisksUsage): SunburstNode => {
+      const transform = (
+        node: HostDisksUsage,
+        parentNode?: HostDisksUsage
+      ): SunburstNode => ({
+        // prefix with parent id to avoid duplicate ids
+        id: parentNode ? `${parentNode.id}.${node.id}` : node.id,
+        name: this._formatLabel(node.id) || node.label,
+        value: node.used_bytes,
+        children: node.children?.map((child) => transform(child, node)),
+      });
+      return transform(storageInfo);
+    }
+  );
+
+  private _formatBytes = (bytes: number): string => {
+    const gb = this._bytesToGB(bytes);
+    return `${roundWithOneDecimal(gb)} GB`;
+  };
+
+  private _formatLabel = (id: string): string =>
+    this.hass.localize(`ui.panel.config.storage.segments.${id}`) || id;
+
+  private _bytesToGB(bytes: number): number {
+    return bytes / 1024 / 1024 / 1024;
+  }
+
+  private _gbToBytes(GB: number): number {
+    return GB * 1024 * 1024 * 1024;
+  }
+
+  static styles = css`
+    .header {
+      display: flex;
+      align-items: flex-start;
+      justify-content: space-between;
+      margin-bottom: 8px;
+    }
+
+    .heading-text {
+      display: flex;
+      flex-direction: column;
+      gap: 4px;
+    }
+
+    .heading {
+      font-weight: 500;
+      font-size: 14px;
+      color: var(--primary-text-color);
+    }
+
+    .description {
+      font-size: 12px;
+      color: var(--secondary-text-color);
+    }
+
+    ha-icon-button {
+      --mdc-icon-button-size: 36px;
+      --mdc-icon-size: 20px;
+      color: var(--secondary-text-color);
+    }
+
+    .chart-container {
+      transition: height 0.3s ease;
```

Comment:
```suggestion
      transition: height var(--ha-animation-base-duration) ease;
```

This is 350ms or 0ms when reduced motion is active